internals/jest/setupTests.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+// react-testing-library renders your components to document.body,
+// this adds jest-dom's custom assertions
+import '@testing-library/jest-dom/extend-expect'
+import * as emotionJestUtils from '@emotion/jest'
+
+import 'react-app-polyfill/stable'
```

Comment:
Why we need polyfills in test env?

Reply:
:shrug: looks like we don't

Reply:
Ah, I'd keep it to reduce debugging in case we use `'a'.replaceAll('a', 'b')` or `[1,2,3].at(-1)`